scripts/us_census/acs5yr/subject_tables/common/prefetched_outputs/Person_dc_props.json

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Can prefetched_outputs be generated by the script? If so, perhaps skip these files or add instruction on how to generate them.

Reply:
prefetched_outputs is used by `acs_spec_generator_test.py`. Have included a note in `scripts/us_census/acs5yr/subject_tables/common/README.md` on how to refresh them.